src/pages/Home/Home.test.jsx

```diff
@@ -0,0 +1,11 @@
+import Button from "../../components/Button/Button";
+import { customRender } from "../../utils/testUtils";
+import { screen } from "@testing-library/react";
+
+it("should render a button on the screen", () => {
+  customRender(<Button buttonImgClass={"discover-icon"} />);
+
+  const buttononscreen = screen.getByRole("button");
+
+  expect(buttononscreen).toBeInTheDocument();
+});
```

Comment:
Why are we testing the button in the Home test? Probably should be testing the `<Home>` component!